frontend/src/app/features/user-preferences/reminder-settings/reminder-time/reminder-settings-daily-time.component.html

```diff
@@ -1,26 +1,27 @@
-<ng-container
-  *ngIf="(dailyReminders$ | async) && (selectedTimes$ | async) as selectedTimes">
-  <!-- TODO: use checkbox component for the enabled input -->
-  <div
-    class="op-reminder-settings-daily-time--enable">
+<ng-container *ngIf="(selectedTimes$ | async) as selectedTimes"
+              [formGroup]="form"
+>
+  <op-checkbox-field
+          [control]="form.get('enabled')"
+  >
     <input
-      [ngModel]="enabled$ | async"
-      (ngModelChange)="toggleEnabled($event)"
-      type="checkbox"
-      id="op-reminder-settings-daily-time--enable">
-    <label
-      for="op-reminder-settings-daily-time--enable"
-      [textContent]="text.enable">
-    </label>
-  </div>
+            slot="input"
+            type="checkbox"
+            formControlName="enabled"
+    />
+    <p slot="description">{{ text.enable }}</p>
+  </op-checkbox-field>
 
   <div *ngFor="let time of selectedTimes; index as i; trackBy: index"
     class="op-reminder-settings-daily-time--row">
+
     <input
+      *ngIf="(activeTimes$ | async) as activeTimes"
       type="checkbox"
       [ngModel]="isActive(time)"
       (ngModelChange)="toggleActive($event, i, selectedTimes)"
-      [disabled]="(activeTimes$ | async).length === 0 || (activeTimes$ | async).length === 1 && (activeTimes$ | async)[0] === time"
+      [ngModelOptions]="{standalone: true}"
+      [disabled]="activeTimes.length === 0 || activeTimes.length === 1 && activeTimes[0] === time"
```

Comment:
Since the async pipe is no longer needed, the code could be placed inside the component.